mobile/src/main.jsx

```diff
@@ -39,10 +39,7 @@ document.addEventListener('DOMContentLoaded', () => {
   localforage.getItem('state').then(state => {
     if (state) {
       store.dispatch({ type: SET_STATE, state })
-      cozy.init({
-        cozyURL: state.serverUrl,
-        offline: {doctypes: ['io.cozy.files'], timer: 15}
-      })
+      init(state.serverUrl)
```

Comment:
I have always this error: https://github.com/cozy/cozy-client-js/blob/master/src/index.js#L180